vagrant/windows/README.md

```diff
@@ -2,15 +2,13 @@
 
 This Vagrant box can be used to test various features like SMB `Previous Versions` which are not available on Linux.
 
-## Requirements
-
 1. Install plugins
 
 ```
-vagrant plugin install winrm
+$ vagrant plugin install winrm
 ```
 
-## Troubleshooting
```

Comment:
Do you mind if I fix this in both? There should only be 1 H1 in a markdown doc

Reply:
In my opinion, this makes no sense here as there is no real main title in these documents.